### ISSUE TITLE
Script support?

### ISSUE THREAD

**Issue by gingerbeardman** [itchio/itch#1299]
It would be great if you could add some sort of script support. 

Even just shelling a script would suffice.

With that I can run Windows games on Mac using my usual setup (CrossOver/Wine)

Happy to discuss further!

**Comment by fasterthanlime** [comment]
I've been considering Wine support (never tried the CrossOver version myself), but I'm curious how you would see it work / what exactly did you mean by "script" ?

**Comment by gingerbeardman** [comment]
My suggestion was just the one that would require the least amount of work for you and would enable me to help myself. 

My usual workflow is to fire up CrossOver, select my "standard" Wine bottle (one that already has MSVC runtime, .net, D3D, etc installed) and open up CrossOver's Run dialog, select the .exe I've just downloaded and press Run. The game (usually) starts. 

If I could get the path of the downloaded and extracted .exe from the itch.io app then I could automate that workflow.

Note: Wineskin Winery is also a good alternative to CrossOver on macOS.

Aside: Mono-compatible games are another interesting set of non-native games that could easily be run in this way.

-- 

If you're willing to go all-in on Wine then everything could be managed by the itch.io app.

My ideal UX would be: exactly the same as installing and launching a native Mac/Web game. 

Behind the scenes a new/existing Wine bottle is being populated and a shortcut "app" created. Given that you already prompt if the user needs to install any prerequisites then the bottle should never have anything missing (my most common issue when I started out using CrossOver). Uninstall would either delete the bottle or remove files from a shared bottle. One bottle weighs a minimum of ~250MB in my experience, so shared bottled might be something to consider.

There would probably have to be some mechanism for developers to mark compatibility, perhaps as another prereq? Reporting broken games would be done the same way.

How seriously are you considering Wine?
